bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/Container.java

```diff
@@ -58,7 +58,7 @@ public interface Container extends Base, BaseConstraint, ContainerExporter {
 
     @JsonIgnore
     @Override
-    default boolean isRequired() {
+    default Boolean isRequired() {
```

Comment:
this will introduce byte code incompatibility in the FormContainer interface being used internally in far. can you verify this with 20th Dec far (by generating DoR)?

Reply:
will check this

Reply:
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "lang" (class com.adobe.aem.forms.af.rest.model.FormSubmitInfoMetaData), not marked as ignorable (one known property: "draftId"])_ at [Source: (String)"{__  "lang": "en"__}"; line: 2, column: 12] (through reference chain: com.adobe.aem.forms.af.rest.model.FormSubmitInfoMetaData["lang"])
	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1127) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2023) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1700) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1678) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:320) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:323) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4674) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3629) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3597) [com.fasterxml.jackson.core.jackson-databind:2.13.4]
	at com.adobe.aem.forms.af.rest.internal.servlets.AdaptiveFormSubmitServlet.createFormSubmitInfoFromRequest(AdaptiveFormSubmitServlet.java:320) [com.adobe.aem.com.adobe.aem.forms.af.rest:0.0.68]
	
	
	dor generation is failing but not due to this change, its failing because of "lang" key according to logs, the af.rest bundle is very old in far, a major version bump has been done since.

Reply:
do you mean dor is already not working on the latest set up?

Reply:
Checked again, this is not failing at this step.  We are sending data like following , which is causing this error log for key "lang".

data: {
  "numberinput1678181709108": "1233"
}
submitMetadata: {
  "lang": "en"
}

Reply:
lets log a bug  for FormSubmitMetaData issue as we should allow json serialisation of any custom prop in this object.